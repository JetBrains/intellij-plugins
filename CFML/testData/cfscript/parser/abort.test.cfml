component {
  function foo() {
    var a = 1;
    abort;
  }
}