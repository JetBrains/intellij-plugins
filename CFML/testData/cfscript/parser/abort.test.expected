CfmlFile:abort.cfml
  COMPONENT_DEFINITION
    PsiElement(COMPONENT_KEYWORD)('component')
    PsiElement(L_CURLYBRACKET)('{')
    FunctionDefinition
      PsiElement(FUNCTION_KEYWORD)('function')
      PsiElement(IDENTIFIER)('foo')
      ParametersList
        PsiElement(L_BRACKET)('(')
        PsiElement(R_BRACKET)(')')
      FUNCTIONBODY
        BLOCK_OF_STATEMENTS
          PsiElement(L_CURLYBRACKET)('{')
          Assignment
            PsiElement(VAR_KEYWORD)('var')
            ReferenceExpression
              PsiElement(IDENTIFIER)('a')
            PsiElement(ASSIGN)('=')
            IntegerLiteral
              PsiElement(INTEGER)('1')
          PsiElement(SEMICOLON)(';')
          PsiElement(ABORT_KEYWORD)('abort')
          PsiElement(SEMICOLON)(';')
          PsiElement(R_CURLYBRACKET)('}')
    PsiElement(R_CURLYBRACKET)('}')